payments-core/src/main/java/com/stripe/android/view/CardMultilineWidget.kt

```diff
@@ -394,8 +392,8 @@ class CardMultilineWidget @JvmOverloads constructor(
         }
 
         cardNumberEditText.possibleCardBrandsCallback = { brands ->
-            val currentBrand = cardBrandView.brand
             cardBrandView.possibleBrands = brands
+            val currentBrand = cardBrandView.brand
```

Comment:
Same here for ordering comment above

Reply:
Removed 👍 . Figured out the state issue that was causing it